skt/kerneltree.py

```diff
@@ -259,8 +259,19 @@ def __get_remote_url(self, remote):
             match = re.match('Fetch URL: (.*)', line)
             if match:
                 return match.group(1)
+        return None
 
     def __get_remote_name(self, uri):
+        """
+        Get the remote name of a specific URI.
+
+        Args:
+            uri: The URI of the repository's remote.
+
+        Returns:
+            A string representing the remote name
+            None if the uri doesn't exist.
```

Comment:
Checking the code, I don't think this method will ever return `None`. Invalid URLs would throw an exception on the parsing and the valid ones will have a value assigned. Do you have an example where `None` can be returned?

Reply:
Yeah. I agree